integration-tests/oauth2-reactive/src/test/resources/testRunner.yml

```diff
@@ -67,4 +67,5 @@ scenarios:
     - --okta.oauth2.scopes=offline_access
 #    - --okta.oauth2.redirectUri=/authorization-code/callback # not implemented until Spring Sec 5.2
     - --server.servlet.session.tracking-modes=cookie
+    - --spring.security.oauth2.client.registration.okta.client-authentication-method=none
```

Comment:
@mraible looks like this is one of the things that changed in the latest version of Spring Security (I've got a question on the Spring Sec Gitter to confirm this)

This prop needs to be set in order for PKCE (for clients without secrets) to work.  Assuming this is correct, any thoughts on pushing this down to the user like this? or should we re-map it to `okta.oauth2.client-authentication-method=none`, or... should we just detect this condition and configure it automatically (to insure the previous functionality)

(PKCE still isn't used for clients with secrets, that that's another thread...)

Reply:
I would configure it automatically, then document it. That way, it's a seamless upgrade for folks. As far as I know, you can configure Spring Boot as a resource server with only an issuer. I'm not sure folks will know that an issuer and a client ID means it'll use PKCE. A lot of our JS SDKs have a `pkce` flag that you can turn off. Does it make sense to add an `okta.oauth2.pkce` property?